ingest-app/src/cmr/ingest/services/granule_bulk_update/opendap/echo10.clj

```diff
@@ -2,28 +2,34 @@
   "Contains functions to update ECHO10 granule xml for OPeNDAP url bulk update."
   (:require
    [clojure.data.xml :as xml]
-   [clojure.string :as string]
+   [clojure.string :as str]
```

Comment:
CMR naming convention is to use `string` rather than `str` as `str` is a function.